pkg/cmd/output/output.go

```diff
@@ -11,8 +11,41 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-var FormatFlag string
-var Output interface{}
+const (
+	FormatFlagDescription = `Output format. Must be one of (yaml, json)`
+	FormatFlagName        = "format"
+	FormatFlagShortHand   = "f"
+)
+
+var (
+	FormatFlag  string
+	standardOut *os.File
+)
+
+type OutputFormatter struct {
+	data      interface{}
```

Comment:
Can you explain why the OutputFormatter struct and removing data as an argument to the Print function is necessary?

Reply:
For this I wanted to encapsulate the formatting process, but if U feel it's not necessary I could revert